SpatialGDK/Source/SpatialGDKTests/Examples/SpatialGDKTestGuidelines.h

```diff
@@ -0,0 +1,69 @@
+/*
+1. How to run tests:
+	- Tests can be run via Session Frontend in UE4 Editor
+	(https://docs.unrealengine.com/en-US/Programming/Automation/index.html has more information)
+
+	- Tests can be run via command line:
+	{PathToUnreal}\Engine\Binaries\Win64\UE4Editor-Cmd.exe ^
+	 "{PathToUProjectFile}.uproject" ^
+	 -unattended -nopause -NullRHI -log -log=RunTests.log ^
+	 -ExecCmds="Automation RunTests {TestFilter}; Quit"
+
+	where {TestFilter} is a match on the test names
```

Comment:
I'm wondering if we can create categories based on the naming pattern "TestGroup.TestSubgroup.Placeholder Test", and run a subset when needed.
One use case for this would be having a specific sub-cathegory for all long-running tests, or stress tests, so that in a standard CI pass they won't be run, while the nightly CI run will run all.

Reply:
This is possible. I expect we'll use that feature at a certain point. There was no need right now since running all the tests is fast enough.